### ISSUE TITLE
[bug] SignedMeasureFormatter (and I guess more pipelines) are broken

### ISSUE THREAD

**Issue by DavidLapous** [DavidLapous/multipers#43]
For instance, if normalize is true here : 
```python
images = torch.rand(10, 50,50,2, dtype=torch.float64).requires_grad_(True)
s = [mp.filtrations.Cubical(img) for img in images]
sms = [mp.signed_measure(_s) for _s in s]
sms = mms.SignedMeasureFormatter(
    unrag=True, deep_format=True, normalize=True
).fit_transform(sms)
```

**Comment by DavidLapous** [comment]
Fixed in latest release.
